### DISCUSSION TITLE
Sync cong server media

### DISCUSSION THREAD

**Discussion by stefano799** [sircharlo/meeting-media-manager#1072] [Q&A] [answered]
google translator

Hi . I have a problem with sync.
in settings i see shared folders and i also see remote uploaded media.
how can I see them in the app and reproduce them on the screen?
I only see them on the ISC settings page.
where should I go to have the files available?

**Comment by mtdvlpr** [comment]
@stefano799, I'm not sure I understand what you're saying, but I think you want to present media. If that's the case, in settings, you need to turn on "present media on external display" under "media setup". Then, you will have a blue play button available.

**Comment by mtdvlpr** [comment]
What language do you speak? There's probably a user out there who speaks the same language who can help you out.

**Reply by stefano799** [reply]
italiano

**Comment by stefano799** [comment]
the problem that i don't see the synced files anywhere and i can't play them.

I only see the folder in the sync settings

<img width="617" alt="image" src="https://user-images.githubusercontent.com/122279621/211312965-d84d744e-e99d-4257-9d8c-2361f3683b51.png">

How do I play files that are in the cloud?

**Reply by mtdvlpr** [reply]
@stefano799, those files are automatically downloaded together with the media from jw.org, but they need to be added in the correct format (which is automatically done if you add media through M3). All media should be in a media/[date] folder where date is the date for which the media is needed in the same format as is configured in the settings.

**Reply by sircharlo** [reply]
Do not add the files to the cloud yourself. Add them using M3. To add a file to the cloud using M3, click on the day (for example, Sunday) and add the file using this screen. Then, it will be in the cloud and you will be able to play it using M3 as well. Hopefully, this makes sense.

**Comment by Chipperchap** [comment, marked as answer]
Hi @stefano799,
Please see video - hope it helps (sorry, I cannot speak Italian) 

https://user-images.githubusercontent.com/35364657/211428409-ec1b8fb1-de79-448e-81d6-a62e41698544.mp4
